testsuite/cluster-test/src/aws.rs

```diff
@@ -36,7 +36,8 @@ pub async fn set_asg_size(
     };
     let credentials_provider = WebIdentityProvider::from_k8s_env();
 
-    let dispatcher = rusoto_core::HttpClient::new().expect("failed to create request dispatcher");
+    let dispatcher = rusoto_core::HttpClient::new()
+        .map_err(|_| anyhow!("Failed to create request dispatcher"))?;
```

Comment:
maybe this allows an early exit rather than wasting time retrying?

Reply:
All three calls to `set_asg_size` are `map_err`-ed or `unwrap_or_else`-ed, which makes me think @andll did not want them to panic out of a failed tls negotiation. There's no retry AFAICT.